### ISSUE TITLE
Update nx-remotecache-custom to 19

### ISSUE THREAD

**Issue by MonstraG** [John98Zakaria/nx-remotecache-gcp#79]
Hi, it's me again)

pnpm install on this:
```
	"devDependencies": {
		"@nx-dotnet/core": "2.2.0",
		"@nx/js": "19.0.6",
		"nx": "19.0.6",
		"nx-remotecache-gcp": "2.4.0"
	},
```

gives 

```
 WARN  Issues with peer dependencies found
└─┬ nx-remotecache-gcp 2.4.0
  └─┬ nx-remotecache-custom 18.0.0
    └── ✕ unmet peer nx@^18.0.0: found 19.0.6
```

`nx-remotecache-custom` 19 expects peer of `nx` 19, so please, update `nx-remotecache-custom`.

(yes i just updated numbers in the description lol)

**Comment by John98Zakaria** [comment]
I released 3.0.0 a few hours ago.

Can you try again?

**Comment by MonstraG** [comment]
Yep, it's cool, closing, thanks!
